java/src/main/java/sh/platform/languages/sample/RabbitMQSample.java

```diff
@@ -0,0 +1,56 @@
+package sh.platform.languages.sample;
+
+import sh.platform.config.Config;
+import sh.platform.config.RabbitMQ;
+
+import javax.jms.Connection;
+import javax.jms.ConnectionFactory;
+import javax.jms.MessageConsumer;
+import javax.jms.MessageProducer;
+import javax.jms.Queue;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import java.util.function.Supplier;
+
+public class RabbitMQSample implements Supplier<String> {
+
+    @Override
+    public String get() {
+        StringBuilder logger = new StringBuilder();
+
+        // Create a new config object to ease reading the Platform.sh environment variables.
+        // You can alternatively use getenv() yourself.
+        Config config = new Config();
+        try {
+            //Get the credentials to connect to the RabbitMQ service.
+            final RabbitMQ credential = config.getCredential("rabbitmq", RabbitMQ::new);
+            final ConnectionFactory connectionFactory = credential.get();
+
+            //Connect to the RabbitMQ server
```

Comment:
Nitpick: Comments should have a leading space after the //, and end in a period.  (Here and elsewhere.)

Reply:
Also, capitalize the first letter.